Introducing SuperMegaAwesomeNinja-man

delneet opened the pull request:
Added introduction file introducting myself

[ticket_url: X]

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

Commit:
Introducing SuperMegaAwesomeNinja-man

Added introduction file introducting myself

[ticket_url: X]

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,3 @@
+Hi there, just trying some things out
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Commit:
SpecifY favorite codeschool path

Specified my favorite codeschool path in the introduction file

[ticket_url: X]

Commit:
Specify Ruby as favorite codeschool path

[ticket_url: X]

Commit:
Specify deadly skills

[ticket_url: X]

Commit:
Specify deadly skills

[ticket_url: X]

Commit:
Specify deadly skills

[ticket_url: X]

Commit:
Specify another special skill

Commit:
Specify another special skill

codeschool-kiddo committed:
Added new rule about coffee

Commit:
Dojo rules spelling corrections

Commit:
Merge pull request #2 from delneet/new_rules

Too many people are spilling coffee

Commit:
Merge branch 'deadly_skills'

Commit:
Merge remote-tracking branch 'github/deadly_skills'

Commit:
Kill bad coding practices

Replaced names on kill list with bad coding practices that should get killed

[ticket_url: X]

Commit:
Merge branch 'release_branch_1.0'

Commit:
Kill long functions
fixes #3
[ticket_url: X]

Commit:
Merge pull request #4 from delneet/kill_list

Kill long functions

Commit:
Update README.md
